src/NuGet.Service.Search.WebSite/Startup.cs

```diff
@@ -53,6 +69,23 @@ public void Configuration(IAppBuilder app)
             app.Run(Invoke);
         }
 
+        private static void TrackTelemetryOnIndexReopened(object sender, SegmentInfoEventArgs e)
+        {
+            var telemetryClient = new TelemetryClient();
+
+            // track the index reopened event
+            var eventTelemetry = new EventTelemetry("Index Reopened");
```

Comment:
I would name this event "IndexReopened", it makes searching easier (no need to add quotes)